tests/driver_ad7746/main.c

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Once we agreed on the interface and the test is adapted accordingly, you should add a short README

Reply:
Added